ee_tests/src/specs/ui/space_dashboard/space_dashboard_header.ts

```diff
@@ -0,0 +1,39 @@
+import { ElementFinder, by } from 'protractor';
+import { Header } from './../../page_objects/app/header';
+import { PlannerPage } from './../../page_objects/planner/planner.page';
+import * as ui from './../../ui';
+import * as support from '../../support';
+
+export class SpaceDashboardHeader extends Header {
+  analyzeTab = new ui.Clickable(
+    this.element(by.cssContainingText('li', 'Analyze')),
+    'Analyze');
+  planTab = new ui.Clickable(
+    this.element(by.cssContainingText('li', 'Plan')),
+    'Plan');
+  createTab = new ui.Clickable(
+    this.$('.persistent-secondary').element(by.cssContainingText('li', 'Create')),
+    'Create');
+
+  constructor(el: ElementFinder, spaceName: string, name: string = 'Space Dashboard page header') {
+    super(el, name);
+    this.spaceName = spaceName;
+  }
+  async ready() {
+    support.debug(' ... check if SpaceDashboardPageHeader is Ready');
```

Comment:
Have you considered using this.debug which will print the className as well?

there is also a better way to doing `ready` which logs better messages when `ready` starts and ends see ui/base.element.ts - ready and run

Reply:
@sthaha I decided to do it similar to https://github.com/fabric8io/fabric8-test/blob/master/ee_tests/src/specs/page_objects/app/header.ts#L14

Do you think we should do it
```
async ready() {
   this.run('... check if SpaceDashboardPageHeader is Ready', async () => {
    ...
    ...
  })
}
``` 
this way?